### PR TITLE
OP-519 use common keys for buttons

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-519.

### CODE REVIEW

File: src/main/java/org/isf/operation/gui/OperationList.java [hunk 1]
```diff
@@ -193,14 +193,14 @@ public void mouseClicked(MouseEvent e) {
 		});
 		panelButtons.add(deleteButton);
 
-		JButton cancelButton = new JButton(MessageBundle.getMessage("angal.operationrowlist.close")); //$NON-NLS-1$
-		cancelButton.addMouseListener(new MouseAdapter() {
+		JButton closeButton = new JButton(MessageBundle.getMessage("angal.operationrowlist.close")); //$NON-NLS-1$
+		closeButton.addMouseListener(new MouseAdapter() {
 			@Override
 			public void mouseClicked(MouseEvent evt) {
 				cancelButtonMouseClicked(evt);
```
Comment (mwithi) [inline src/main/java/org/isf/operation/gui/OperationList.java:200]: this method maybe need refactoring

File: src/main/java/org/isf/lab/gui/ExamPicker.java [hunk 2]
```diff
@@ -279,7 +279,8 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
             }
         });
 
-        jButtonQuit.setText(MessageBundle.getMessage("angal.common.cancel"));
+	    jButtonQuit.setText(MessageBundle.getMessage("angal.common.cancel.btn"));
+	    jButtonQuit.setMnemonic(MessageBundle.getMnemonic("angal.common.cancel.btn.key"));
```
Comment (mwithi) [inline src/main/java/org/isf/lab/gui/ExamPicker.java:283]: this button maybe needs refactoring as well

File: src/main/java/org/isf/hospital/gui/HospitalBrowser.java [hunk 3]
```diff
@@ -250,12 +250,12 @@ public void saveConfirm() {
...
-			editJButton = new JButton(MessageBundle.getMessage("angal.common.edit"));
-			editJButton.setMnemonic(KeyEvent.VK_E);
+			editJButton = new JButton(MessageBundle.getMessage("angal.common.edit.btn"));
+			editJButton.setMnemonic(MessageBundle.getMnemonic("angal.common.edit.btn.key"));
 			updateJButton = new JButton(MessageBundle.getMessage("angal.hospital.update"));
 			updateJButton.setMnemonic(KeyEvent.VK_U);
-			exitJButton = new JButton(MessageBundle.getMessage("angal.common.close"));
-			exitJButton.setMnemonic(KeyEvent.VK_C);
+			exitJButton = new JButton(MessageBundle.getMessage("angal.common.close.btn"));
+			exitJButton.setMnemonic(MessageBundle.getMnemonic("angal.common.close.btn.key"));
```
Comment (mwithi) [inline src/main/java/org/isf/hospital/gui/HospitalBrowser.java:258]: this button maybe needs refactoring as well

File: src/main/java/org/isf/dicom/gui/DicomGui.java [hunk 4]
```diff
@@ -205,7 +205,8 @@ private void initComponents() {
 		jButtonDeleteDicom.setName("jButtonDeleteDicom");
 		jButtonDeleteDicom.setEnabled(false);
 
-		jButtonExit.setText(MessageBundle.getMessage("angal.common.close"));
+		jButtonExit.setText(MessageBundle.getMessage("angal.common.close.btn"));
+		jButtonExit.setMnemonic(MessageBundle.getMnemonic("angal.common.close.btn.key"));
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/gui/DicomGui.java:209]: Exit or Close is the same, maybe needs refactoring like others

**Review by mwithi — APPROVED** [review]
Just few comments, othrwise LGTM. Thanks a lot!

File: src/main/java/org/isf/dicom/gui/DicomGui.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/gui/DicomGui.java]: Thanks; done.

File: src/main/java/org/isf/hospital/gui/HospitalBrowser.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/hospital/gui/HospitalBrowser.java]: Thanks; done.

File: src/main/java/org/isf/lab/gui/ExamPicker.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/ExamPicker.java]: Thanks; done.

File: src/main/java/org/isf/operation/gui/OperationList.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/operation/gui/OperationList.java]: Thanks; done.

**Pull request merged**